app/governance/[id]/page.tsx

```diff
@@ -0,0 +1,137 @@
+"use client";
+import Text from "@/components/text";
+import styles from './proposalModal.module.scss';
+import useSingleProposalData from "@/hooks/gov/useSingleProposalData";
+import { calculateVotePercentages, formatDeposit, formatProposalStatus, formatProposalType, formatTime } from "@/utils/gov/formatData";
+import Icon from "@/components/icon/icon";
+import Button from "@/components/button/button";
+import useProposals from "@/hooks/gov/useProposals";
+
+export default function Page({ params }: any) {
+  const {proposals} = useProposals(
+    {chainId: 7700}
+  )
+  
+  
+  if(!proposals || proposals.length==0){
+    return (
+      <div>Loading Proposals....</div>
+    );
+  }
+  const proposal = proposals.find((p) => p.proposal_id === params.id);
+
+  if (!proposal) {
+    return <div>No proposal found with the ID {params.id}</div>;
+  }
+  console.log(proposal);
+  const isActive = formatProposalStatus(proposal.status)=='ACTIVE';
+
+  const votesData = calculateVotePercentages(proposal.final_tally_result);
+
+  return (
+  <div className={styles.proposalContainer}>
+    <div className={styles.proposalHeaderContainer}>
+        <div className={styles.proposalCard1}>
+          <div style={{borderRight: "1px solid", padding: "10px"}}><Text>#{proposal.proposal_id}</Text></div>
+          <div style={{padding: "10px"}}><Text>{formatProposalStatus(proposal.status)}</Text></div>
+        </div>
+        <div>
+            <Text font='proto_mono' size="x-lg">{proposal.content.title}</Text>
+        </div>
+        <div>
+          <Text opacity={0.4}>{proposal.content.description}</Text>
+        </div>
+    </div>
+    
+    <div className={styles.proposalCardContainer}>
+      <div className={styles.proposalCardContainer1}>
+        <div className={styles.proposalInfoBox}>
+          <div className={styles.proposalInfo}>
+            <div><Text font="proto_mono" opacity={0.3}>Type:</Text></div>
+            <div><Text font="proto_mono">{formatProposalType(proposal.content["@type"])}</Text></div>
+          </div>
+          <div className={styles.proposalInfo}>
+            <div><Text font="proto_mono" opacity={0.3}>Total Deposit:</Text></div>
+            <div>
+              <Text font="proto_mono">{formatDeposit(proposal.total_deposit[0].amount)} </Text>
+            </div>
+          </div>
+          <div className={styles.proposalInfo}>
+              <div><Text font="proto_mono" opacity={0.3}>Turnout/ Quorum: </Text></div>
+              <div><Text font="proto_mono">38.1% 33.4%</Text></div>
```

Comment:
Make variables for these since they could change. Will be easier to update instead of hardcoding here